src/namespaces.json

```diff
@@ -198,5 +198,24 @@
     "min": 0,
     "invalid": [],
     "visible": false
+  },
+  "unidao": {
+    "key": "unidao",
+    "symbol": "MRQ",
```

Comment:
Isn't the symbol and name supposed to be: MarsReputation (MRR)?

Reply:
Yes, correct! Thank you a lot!

Reply:
MRR is one of the reputation tokens in UniDAO Ecosystem, currently used for governance. So, yes, it's correct name (UniDAO), but incorrect naming,